src/devices/tuya.ts

```diff
@@ -794,6 +804,20 @@ const modernExtendLocal = {
 };
 
 const definitions: Definition[] = [
+    {
+        fingerprint: [
```

Comment:
Doesn't this device work out of the box? It should be detected as https://github.com/Koenkk/zigbee-herdsman-converters/blob/1f1c553b9ce1c6bab4f53f365c0feba7cf66515b/src/devices/tuya.ts#L6820, for proper detection you can add a `tuya.whiteLabel` there (search `tuya.ts` for examples)

Reply:
You're right. Sorry, my bad.

With zigbee2mqtt 1.36 this device had problems. I didn't realize that it was fixed in later versions.
Sorry again.

I'll try to add a tuya.whiteLabel for model FS-05R (I already added a photo to docs' repository).